python/arcticdb/version_store/_store.py

```diff
@@ -482,6 +483,8 @@ def write(
             Removes previous (non-snapshotted) versions from the database.
         pickle_on_failure: `bool`, default=False
             Pickle `data` if it can't be normalized.
+        ensure_sorted: bool, default=False
+            If True, will verify new data is sorted, but only if it contains a datetime index.
```

Comment:
Am I right in saying:

- We are just using `is_monotonic_increasing`/`is_monotonic_decreasing` from Pandas for this verification, we are not doing any of our own checks if Pandas does not know?
- We will not reorder unsorted data, it is only a check?

If so, I think it is worth being explicit in the docstring about both of these, as well as specifying what exception will be thrown if this flag is true but unsorted data is provided.

Reply:
yes. you are right with both. 

For your suggestion: regarding your second point I thing we already are explicit on that in the docstring.  Your first point is fair to add in the doctoring in a similar way we are already doing in the exception message.